tests/test_execution_compute_plan.py

```diff
@@ -325,7 +325,6 @@ def test_execution_compute_plan_canceled(global_execution_env):
 
     traintuples = cp.list_traintuple()
```

Comment:
I think we should do a `cp.wait()` before the end of the test.

Unfortunately it is currently the responsibility of the test to return after all running tuples have been completed. This is required to ensure that a new test will start with the substra network ready to execute new tuples.

Reply:
Just to be sure you think it should be added  at the bottom at the test, after the assertions, right ?

Reply:
It could be added:
- before the `list_traintuples()` and then we should check the status are all in a final state canceled or done
- as the last line of the tests